### PR TITLE
[KOGITO-4063] - Update nightly and PR check image with operator-sdk 1.2

### PR DESCRIPTION
**Pull request by Kaitou786**
See: https://issues.redhat.com/browse/KOGITO-4063

Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

### CODE REVIEW

File: kogito-cloud-operator-jenkins-slave/image.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@ modules:
 packages:
   content_sets:
     x86_64:
-      - rhel-7-server-extras-rpms
-      - rhel-7-server-rpms
+      - rhel-8-for-x86_64-baseos-rpms
+      - rhel-8-for-x86_64-appstream-rpms
```
Comment (Kaitou786) [inline kogito-cloud-operator-jenkins-slave/image.yaml:25]: The base image we use `registry.redhat.io/openshift4/ose-jenkins-agent-base` now uses a rhel8 based base image, so needed to update the content sets :)

**Comment by Kaitou786** [comment]
Can be merged in coherence to https://github.com/kiegroup/kogito-cloud-operator/pull/695

**Comment by radtriste** [comment]
quick note: this is only for PR check or ?
the one for nightly should be updated https://github.com/kiegroup/kie-jenkins-scripts or ?

**Review by radtriste — APPROVED** [review]

**Comment by Kaitou786** [comment]
> quick note: this is only for PR check or ?
> the one for nightly should be updated https://github.com/kiegroup/kie-jenkins-scripts or ?

Yes, it is only for the PR check see the accompanying PR for the nightly: https://github.com/kiegroup/kie-jenkins-scripts/pull/881
(I need to test the image first before opening it for review :) )

**Review by spolti — APPROVED** [review]

**Pull request merged**